Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -317,7 +317,11 @@ bool areEncryptedCaps(const GstCaps* caps)
         GST_WARNING("caps are empty");
         return false;
     }
-    return gst_structure_has_name(structure, "application/x-cenc") || gst_structure_has_name(structure, "application/x-webm-enc");
+    return gst_structure_has_name(structure, "application/x-cenc") || gst_structure_has_name(structure, "application/x-webm-enc")
+#if ENABLE(CBCS)
```

Comment:
add "application/x-cbcs" in between other conditions to not have odd line only with ;